xesmf/frontend.py

```diff
@@ -272,6 +273,17 @@ def __init__(
             If not given or if those are not found on the regridded object, regridding
             uses the two last dimensions of the object (or the last one for input LocStreams and Meshes).
 
+        unmapped_to_nan: boolean, optional
+            Option to select the default value for undefined weights (weights of unmapped cells).
```

Comment:
```suggestion
            Set values of unmapped points to `np.nan` instead of zero (ESMF default). This is useful for target cells lying outside of the source domain when no output mask is defined. If an output mask is defined, or regridding method is `nearest_s2d` or `nearest_d2s`, this option has not effect.
```

Reply:
I've tried to summarize lines 277-284 to make it less lenghty